service/src/main/resources/application.yml

```diff
@@ -72,6 +72,12 @@ management:
     web:
       exposure:
         include: "prometheus"
+  metrics:
+    distribution:
+      # Used to publish a histogram suitable for computing aggregable (across dimensions) percentile
+      # latency approximations in Prometheus (by using histogram_quantile)
+      # For more information: https://micrometer.io/docs/concepts#_histograms_and_percentiles
+      percentiles-histogram[http.server.requests]: true
 
```

Comment:
Some extra info on what's possible here:

We can narrow down the metrics we collect by setting lower and/or upper bounds on the default buckets collected or by picking our own buckets based on established SLAs.  But Jack tells me that these metrics are cheap, and I see that in dire times average workspace resolution latency has taken minutes:

<img width="841" alt="Screenshot 2024-01-12 at 5 53 10 PM" src="https://github.com/DataBiosphere/terra-workspace-manager/assets/79769153/0653eda7-4663-44ae-96df-86ee2199b609">

So I'm sticking with the defaults, and we can tune collection later if desired.